js/home_page.js

```diff
@@ -0,0 +1,47 @@
+/*Test function: can be removed or modified by code reviewer.*/
+document.addEventListener("click", () => {
```

Comment:
This doesn't work, you have to press twice to enter all ingredients. The test should also not be in the final commit